examples/pylint/W1303_missing_format_argument_key.py

```diff
@@ -1,2 +1,3 @@
 # Error on the following line: missing format argument for 'age'
-s = '{last_name}, {fist_name} - {age}'.format(last_name='bond', first_name='james')
+s = '{last_name}, {fist_name} - {age}'.format(
+    last_name='bond', first_name='james')
```

Comment:
The arguments were wrapped on a new line indented with 4 spaces to comply with the 80 char limit. This was undone.